genesis/process/memoryComponents.go

```diff
@@ -14,7 +14,8 @@ func createAccountAdapter(
 	accountFactory state.AccountFactory,
 	trieStorage data.StorageManager,
 ) (state.AccountsAdapter, error) {
-	tr, err := trie.NewTrie(trieStorage, marshalizer, hasher)
+	maxTrieLevelInMemory := uint(5)
```

Comment:
make a constant for this - some component should be able to return the desired level.

Reply:
Done.